quil-rs/src/parser/command.rs

```diff
@@ -120,6 +123,61 @@ pub(crate) fn parse_declare<'a>(input: ParserInput<'a>) -> InternalParserResult<
     ))
 }
 
+/// Parse the contents of a `CALL` instruction.
+///
+/// Note, the `CALL` instruction here is unresolved; it may only be resolved within the
+/// full context of a program. See [`crate::Program::resolve_call_instructions`].
```

Comment:
```suggestion
/// Note, the `CALL` instruction here is unresolved; it can only be resolved within the
/// full context of a program. See [`crate::Program::resolve_call_instructions`].
```